src/Diagnostics.DataProviders/DataProviders.cs

```diff
@@ -7,21 +7,21 @@ public class DataProviders
     {
         private OperationDataCache _cache = new OperationDataCache();
 
-        public IKustoDataProvider Kusto;
-        public ISupportObserverDataProvider Observer;
-        public IGeoMasterDataProvider GeoMaster;
-        public IAppInsightsDataProvider AppInsights;
-        public IChangeAnalysisDataProvider ChangeAnalysis;
-        public IAscDataProvider Asc;
-        public Func<MdmDataSource, IMdmDataProvider> Mdm;
+        public IKustoDataProvider Kusto { get; }
```

Comment:
Please change these back to fields and not properties otherwise detector would fail because they would need to be recompiled. We ran into this before and it was a sad day :(